### PR TITLE
Ng1 ng2 small fixes

### PR DESCRIPTION
**Pull request by CarlyLanglois**
small fixes to the first 2 ng chapters!

also fyi this branch has the version-pinned angular install instructions

### CODE REVIEW

File: src/chapters/angular-lsn1/components.rst [hunk 1]
```diff
@@ -108,6 +108,10 @@ Creating a ``task-list`` component looks something like this:
 From the output, we see that the ``ng generate`` command created four new files
 in the ``src/app/task-list`` folder.
 
+.. note::
+
+   ``ng generate`` can be shortened to ``ng g``.
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/angular-lsn1/components.rst]: honestly I missed that!

File: src/chapters/angular-lsn2/studio.rst [hunk 2]
```diff
@@ -149,6 +149,7 @@ so that the button appears next to each item in the crew list.
 
    <li *ngFor="let member of crew">
       {{member.name}}
+      <span *ngIf="member.firstMission">- 1st</span>
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/angular-lsn2/studio.rst]: lets chat offline

File: src/chapters/more-on-functions/functions-as-values.rst [hunk 3]
```diff
@@ -55,7 +55,7 @@ When a variable refers to a function, we can use the variable name to *call* the
 
 The variable ``helloFunc`` can be thought of as an *alias* for the function ``hello``. When we call the function ``helloFunc``, JavaScript sees that it refers to the function ``hello`` and calls that *specific* function. 
 
-When we use a variable *name*, we are really using its *value*. If the variable ``class`` is assigned the value ``"LC101"``, then ``console.log(class)`` prints ``"LC101"``. When a variable holds a function, it behaves the same way as when it holds a number or a string. The variable *refers to* the function. 
+When we use a variable *name*, we are really using its *value*. If the variable ``course`` is assigned the value ``"LC101"``, then ``console.log(course)`` prints ``"LC101"``. When a variable holds a function, it behaves the same way as when it holds a number or a string. The variable *refers to* the function.
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/more-on-functions/functions-as-values.rst]: yeah!

File: src/chapters/angular-lsn2/studio.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/angular-lsn2/studio.rst]: adding more explicit instructions to for students to add this line - per our convo!

File: src/chapters/angular-lsn2/studio.rst [hunk 4]
```diff
@@ -185,6 +186,7 @@ We need to add a click event to the member name.
 
    <li *ngFor="let member of crew">
       <span (click)="edit(member)" class="editable-text">{{member.name}}</span>
+      <span *ngIf="member.firstMission">- 1st</span>
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/angular-lsn2/studio.rst]: same as above!

File: src/chapters/more-on-functions/functions-as-values.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/more-on-functions/functions-as-values.rst]: OK.

**Pull request merged**